site/en/meet-the-team/events/devfest-nantes/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: DevFest Nantes
+summary: >-
+  The Devfest (Developers Festival), is a technical conference destined to
+  developers. It's meant for students, professionals, or any curious techie.
+image: image/fuiz5I8Iv7bV8YbrK2PKiY3Vask2/dZBRyZIZ0xhHvDtI6eql.png
+location: 'Nantes, France'
+date: '2022-10-20'
+externalUrl: 'https://devfest2022.gdgnantes.com/'
+algoliaUrl: /en/meet-the-team/#devfest-nantes
```

Comment:
This needs to be automatically generated in an 11tydata file because authors will not know how to add this value.